### ISSUE TITLE
alpha-linolenic acid metabolic process (R-HSA-2046106)

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#335]
- [ ] Make the mouse model: http://noctua.geneontology.org/editor/graph/gomodel:675b862c00001162?

**Comment by deustp01** [comment]
> * I think the acid-conjugate base issue creeps up quite a bit in this pathway. They should all be checked.

Yes, definitely. I've smuggled the plan to fix it into the getting-rid-of-molecular-events project. This exact pathway has none but a closely related one does, so I bundled everything together here: "[Linoleic acid (LA) metabolism et al (R-HSA-2046104,5,6) notes](https://docs.google.com/document/d/1rmQGm0QR0IBIHvAY-ATDL11h95GZtEwnQPXCfXDdyms/edit?tab=t.0)"

**Comment by ukemi** [comment]
> This exact pathway has none but a closely related one does

It seems like most of the major players in this pathway are not the correct species: ChEBI[27432](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27432), ChEBI[51985](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:51985), ChEBI[63545](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:63545), ChEBI[63542](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:63542) etc. rather than [27432](http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27432), [CHEBI:74034](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI%3A74034) , [CHEBI:71489](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI%3A71489), [CHEBI:74265](https://www.ebi.ac.uk/chebi/searchId.do;jsessionid=3EEA998803600334EB159CA9F7887C33?chebiId=CHEBI%3A74265)

And you are right, this seems to be an issue with the next pathway I will convert to mouse, R-HSA-2046105. I was going to put that in the ticket I make for that pathway when I start on it. If you would like to tackle them both (makes sense), we should probably open a ticket for that pathway now for record keeping.

**Comment by ukemi** [comment]
Sorry @deustp01. I now realize that you were referring to molecular events.

**Comment by deustp01** [comment]
But a GitHub ticket to cove all aspects of the Reactome cleanup and the mouse GO-CAMs sound good.

**Comment by ukemi** [comment]
Done https://github.com/geneontology/pathways2GO/issues/336
